src/components/registerV2/confirmPassphrase.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import fillWordsList from 'bitcore-mnemonic/lib/words/english';
+import { translate } from 'react-i18next';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { FontIcon } from '../fontIcon';
+import { PrimaryButtonV2, SecondaryButtonV2 } from '../toolbox/buttons/button';
+import registerStyles from './registerV2.css';
+import styles from './confirmPassphrase.css';
+import lock from '../../assets/images/icons-v2/lock.svg';
+import Options from './confirmPassphraseOptions';
+
+class ConfirmPassphrase extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      numberOfOptions: 3,
+      numberOfWords: 2,
+      words: [2, 9],
+      tries: 0,
+      maxTries: 3,
```

Comment:
`numberOfOptions`, `numberOfWords`, `maxTries` doesn't have to be on the state, since it doesn't change.